parachains/runtimes/assets/westmint/src/lib.rs

```diff
@@ -238,6 +269,8 @@ impl pallet_assets::Config for Runtime {
 	type Extra = ();
 	type WeightInfo = weights::pallet_assets::WeightInfo<Runtime>;
 	type AssetAccountDeposit = AssetAccountDeposit;
+	#[cfg(feature = "runtime-benchmarks")]
+	type Helper = XcmBenchmarkHelper;
```

Comment:
@joepetrowski 
please, see: https://github.com/paritytech/cumulus/commit/25c2fc5bbb8a99963928aa20a7029d79604ef453#r95961973

and other instance has only `type Helper = ();` ?

Reply:
`()` assumes that you can cast `u32` into the `AssetId`. We need to mock something for the case where we have `MultiLocation` as an `AssetId`.